scripts/lib/setup_venv.py

```diff
@@ -252,6 +252,37 @@ def setup_virtualenv(target_venv_path, requirements_file, virtualenv_args=None,
     exec(open(activate_this).read(), {}, dict(__file__=activate_this))
     return cached_venv_path
 
+
+def add_cert_to_pipconf():
+    # type: () -> None
+    conffile = os.path.expanduser("~/.pip/pip.conf")
+    confdir = os.path.expanduser("~/.pip/")
+    if not os.path.exists(confdir):
+        os.makedirs(confdir)
+    if os.path.exists(conffile):
+        newfile = os.path.expanduser("~/.pip/pip.conf.new")
+        global_exists = False
+        with open(newfile, 'w') as new:
+            with open(conffile, 'r') as old:
+                for line in old:
+                    if line.startswith('[global]'):
+                        global_exists = True
+                        new.write(line)
+                        new.write("cert = {}\n".format(os.environ.get('CUSTOM_CA_CERTIFICATES')))
+                    elif line.startswith('cert'):
+                        continue
+                    else:
+                        new.write(line)
+                if not global_exists:
+                    new.write('\n[global]\n')
+                    new.write("cert = {}\n".format(os.environ.get('CUSTOM_CA_CERTIFICATES')))
+        os.rename(newfile, conffile)
+    else:
+        with open(conffile, 'x') as new:
+            new.write('[global]\n')
+            new.write("cert = {}\n".format(os.environ.get('CUSTOM_CA_CERTIFICATES')))
```

Comment:
This is a lot of code to get right.  `pip.conf` is in the standard `.ini` format supported by Python's ConfigParser library, so we could use that, but I think this function can be replaced with a single call to `crudini`, which we already install as one of our `apt` dependencies and thus should be available here.

Reply:
I searched for a library but didn't land on ConfigParser or anything alike, thanks for the tips. I will check `crudini` and rewrite as needed.

> This is a lot of code to get right

Indeed, I hand-tested each branch